import

```diff
@@ -332,16 +332,20 @@ var onDataHandler = function(obj) {
 csv
   .fromStream(process.stdin, {headers:true})
   .validate(validate)
-  .on("data", onDataHandler)
-  .on("end", function() {
+  .on('data', onDataHandler)
+  .on('end', function() {
     console.info(
-      "processing %s rows, will take about %s minutes.",
+      'processing %s rows, will take about %s minutes.',
       stats.rows,
       (((stats.rows * wait)/1000) / 60).toFixed(1)
-    )
-  })
+    );
+  });
 
-process.on("exit", function() {
- console.info("");
- console.info(stats);
+process.on('exit', function() {
+  console.info('');
+  console.info(stats);
+
+  if (stats.responses && (!stats.responses['200'] || Object.keys(stats.responses).length > 1)) {
```

Comment:
Only problem here is we get 200 and 201s from couchdb that are success.  So we might test for 201 also or have a for loop that does a `if (retval < 200 || retval > 299) { ... }` or something.